modules/network-observability-working-with-connections.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-working-with-conversations_{context}"]
+= Working with conversation tracking
+As an administrator, you can group network flows that are part of the same conversation. A conversation is defined as a grouping of peers that are identified by their IP addresses, ports, and protocols resulting in a unique *Conversation Id*. Events that are associated with the conversation are queryable in the web console. These events are represented in the web console as follows:
+
+** *Conversation start* happens when an unknown connection is starting or TCP flag intercepted
+** *Conversation tick* occurs at every specified interval defined in flow collector `spec.processor.conversationHeartbeatInterval` while the connection is up.
+** *Conversation end* occurs when the `FlowCollector` `spec.processor.conversationEndTimeout` is reached or TCP flag intercepted.
+** *Flow* is the network traffic flow that occurs within the specified interval.
+
+[NOTE]
+====
+Storage is impacted by enabling this option.
+====
+
+.Procedure
+
+. Configure the `FlowCollector` so that `spec.processor.logTypes`, `conversationEndTimeout`, and `conversationHeartbeatInterval` are set as follows:
++
+[id="network-observability-flowcollector-configuring-conversations_{context}"]
+.Configure `FlowCollector` for conversation tracking
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+ processor:
+  conversationEndTimeout: 10s               <1>
+  logTypes: ALL                  <2>
+  conversationHeartbeatInterval: 30s           <3>
+----
+<1> The *Conversation end* event represents the point when the `conversationEndTimeout` is reached or the TCP flag is intercepted.
+<2> When `logTypes` is set to `ALL`, the *Conversation start*, *Heartbeat*, *Conversation end*, and *Flow* events are exported and visible in the *Network Traffic* page. You can also set the value to `FLOWS` to only export the *Flow* event.
```

Comment:
This needs to be updated too right? Regarding options being `CONVERSATIONS`, `ENDED-CONVERSATIONS`, `FLOWS` and `ALL`

Reply:
Yeah...I think I need to flip around some details that were previously in Viewing the FlowCollector. I added them here instead and left more minimal details in the Viewing FlowCollector topic.